src/leo-client-app/src/components/Scheduler.tsx

```diff
@@ -1,165 +1,204 @@
-                <button
-                  key={index}
-                  className="scheduleButton"
-                  onClick={() => addCommand(command)}>
-                  {command}
-                </button>
+    <Box className="schedulesPageContainer" sx={{backgroundColor: "var(--material-theme-sys-dark-background)", padding: "20px" }}>
+      <Box px = {"200px"}>
+        <SatelliteName noradId="55098" />
+      <Typography className="headerBox2">All Schedules</Typography>
+      <Typography className="headerBox3">Schedule Queue</Typography>
+      </Box>      
+      <Box className="main-schedule"> 
+      <Stack alignItems="flex-start" spacing={1}>
+        {isLoading ? (
+          <Box className="loadingBox">
+            <CircularProgress />
+          </Box>
+        ) : (
+          <Grid
+            className="futureSchedulesBox"
+            container
+            spacing={2}
+            sx={{
+              display: "flex",
+              boxSizing: "border-box",
+              "& .MuiGrid-item": {
+                flex: "0 0 auto",
+              },
+              border: 3,
+              borderRadius: "24px",
+              borderColor: 'white',
+              width: '85%',
+              mx: -2,
+            }}>
+            {scheduleForCard &&
+              scheduleForCard.map((schedule, index) => (
+                <Grid item key={index} sx={{width:'98%'}} >
+                  <Card sx={{
+                  width: '99%',
+                  minHeight: 100,
+                  margin: 0.5,
+                  backgroundColor:
+                    "var(--material-theme-sys-light-primary-container)",
+                  cursor: "pointer",
+                  borderRadius: 4,
+                  }}>
+                    <CardContent>
+                      <Stack spacing={1}>
+                        <Typography className="cardTitle"> 
+                          {formatDate(schedule.startDate)}
+                        </Typography>
+                        <Typography className="cardSubtitle" >
+                          {formatTimeRange(
+                            schedule.startDate,
+                            schedule.endDate
+                          )}
+                        </Typography>
+                        <>
+                          {scheduleCommands[schedule.id] &&
+                          scheduleCommands[schedule.id].length > 0 ? (
+                            scheduleCommands[schedule.id].map(
+                              (commandObj: any, cmdIndex) => (
+                                <Typography key={cmdIndex} className="cardSubtitle">
+                                  {commandObj.command}
+                                </Typography>
+                              )
+                            )
+                          ) : (
+                            <Typography className="cardSubtitle" sx={{padding: "0px", fontSize: "15px", color: "var(--material-theme-black)"}}>No commands</Typography>
```

Comment:
should be replaced by getting the commands and then showing like 3/4 of them, just like in `UpcomingSchedules`

Reply:
fixed, would prefer if you can take another look